docs/developer/discounts/promotions.mdx

```diff
@@ -162,6 +209,54 @@ Below, you can find examples of promotion rules with catalogue predicates that d
 }
 ```
 
+### Order predicate
+
+The order predicates enable the definition of conditions for checkout and draft order objects based on their attributes.
+Currently, it is only possible to define conditions based on the base total and subtotal price.
+
+Catalogue and order promotion discounts can be combined. Therefore, during checkout or draft order qualification,
+the discounted price (after applying any catalogue discount) is utilized.
+
+### Order discount examples
+
+- **10%** discount applied when the subtotal price is above **$20** applied on subtotal price
+
+```json
+{
+  "promotionRule": {
+    "orderPredicate": {
+      "discountedObjectPredicate": {
+        "baseSubtotalPrice": {
+          "range": { "gte": 20 }
+        }
+      }
+    },
+    "rewardValueType": "PERCENTAGE",
+    "rewardValue": 10,
+    "rewardType": "SUBTOTAL_DISCOUNT"
+  }
+}
+```
+
+- **$5** discount applied when the total price is above **$50** applied on subtotal price
+
+```json
+{
+  "promotionRule": {
+    "orderPredicate": {
+      "discountedObjectPredicate": {
+        "baseTotalPrice": {
```

Comment:
btw. Shouldn't be `baseSubtotalPrice`?

Reply:
No, why?
The case is:
`**$5** discount applied when the total price is above **$50** applied on subtotal price`